### PR TITLE
Implemented filtering for events scene

### PR DESCRIPTION
**Pull request by Lainera**

### CODE REVIEW

File: js/scenes/Events/EventsContainer.js [hunk 1]
```diff
@@ -19,6 +20,35 @@ class EventsContainer extends Component {
       }
     }
   }
+  constructor(props) {
```
Comment (codedavinci) [inline js/scenes/Events/EventsContainer.js]: `constructor` not being used, you can remove it all together.

**Review by codedavinci — COMMENTED** [review]
To be adjusted

**Comment by Lainera** [comment]
fixed with af65e77

**Review by codedavinci — APPROVED** [review]
LGTM !

**Pull request merged**